app/controllers/graphql_controller.rb

```diff
@@ -0,0 +1,43 @@
+class GraphqlController < ApplicationController
+  def execute
+    variables = ensure_hash(params[:variables])
+    query = params[:query]
+    operation_name = params[:operationName]
+    context = {
+      # Query context goes here, for example:
+      # current_user: current_user,
+    }
+    result = CrowdhoundBeSchema.execute(query, variables: variables, context: context, operation_name: operation_name)
+    render json: result
+  rescue => e
```

Comment:
We'll have to hit the production server and see what we get back, it seems like GraphQL is handling an error in development differently than in production.